en/docs/tutorials/scenarios/scenario10-notifications-webhooks.md

```diff
@@ -1,4 +1,62 @@
 # Scenario 10 - Notifications Using Webhooks
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**From time to time GoGo transit provides notifications to the Train companies. (For eg: platform unavailability, revised fees etc). GoGo transit is planning to provide these information as webhooks so that the Train companies can subscribe to these notifications without continuously polling.**
```

Comment:
```suggestion
From time to time GoGo transit provides notifications to the Train companies. (For e.g., platform unavailability, revised fees, etc.). GoGo transit is planning to provide this information as webhooks so that the Train companies can subscribe to these notifications without continuously polling.
```